### DISCUSSION TITLE
Skipping challenge, Really bad day secenario.

### DISCUSSION THREAD

**Discussion by Xindage** [cataclysmbn/Cataclysm-BN#3243] [General]
### Describe the bug

Choosing to start at the golf mid course on really bad day start, let you start in a very safe place and make it lost all the challenge that this scenario was supposed to be.

### Steps To Reproduce

1. Start a new character.
2. Choose really bad day scenario.
3. Choose starting location as golf club.
4. Proceed to start the game.
5. Have a free start. (No burning building and no nearby zombies.)

### Screenshots

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/6380147/4b4aeeec-ed82-4b68-b7cc-bfe65de04d6a)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 78572b4 [32-bit]
- Graphics Version: Tiles
- Game Language: English [en_US]
- Mods loaded: [
    Bright Nights [bn],
    Disable NPC Needs [no_npc_food],
    Simplified Nutrition [novitamins],
    No Rail Stations [No_Rail_Stations],
    Elevated bridges [elevated_bridges],
    Aftershock [aftershock]
]

**Comment by chaosvolt** [comment]
So, part of this feels like a bug to me. One of the starting locations starts you on a course it seems, so the lack of burning building is reasonable there, but the other is a clubhouse it seems? That should logically start you inside it and have it start on fire, so if it's not doing that then it's a bug rather than just a balance issue.

**Comment by Krwak** [comment]
Should be likely closed since #3020 was merged already.
